recipes/config.rb

```diff
@@ -26,11 +26,18 @@
   action :delete
 end
 
+# disable the old bamboo-server service
+service 'bamboo-server' do
+  provider Chef::Provider::Service::Upstart
+  action [:disable, :stop]
+end
```

Comment:
Yeah, I remove the ignore_failure and set a only_if block to run only on Centos 6.